packages/fxa-auth-server/test/scripts/prune-tokens.js

```diff
@@ -61,8 +62,8 @@ describe('#integration - scripts/prune-tokens', () => {
     Date.now() -
     moment.duration(maxAge, 'days').asMilliseconds() -
     moment.duration(1, 'day').asMilliseconds();
-  const uid = uuidTransformer.to('f9916686c226415abd06ae550f073cea');
-  const email = 'user1@test.com';
+  const uid = uuidTransformer.to(crypto.randomBytes(16).toString('hex'));
+  const email = `${Math.random}blue@test.com`;
```

Comment:
I think you wanted `Math.random()`? And perhaps should use web crypto to avoid the annoying git hub warnings.